src/dmd/doc.d

```diff
@@ -1905,6 +1918,36 @@ unittest {
     assert(skipChars(&buf, 12, "dne") == 15);
 }
 
+/************************************************
+ * Get the indent from one index to another, counting tab stops as four spaces wide
+ * per the Markdown spec.
+ * Params:
+ *  buf           = an OutBuffer containing the DDoc
+ *  from          = the index within `buf` to start counting from, inclusive
+ *  to            = the index within `buf` to stop counting at, exclusive
```

Comment:
Why is there so much space here? If you're going to align the parameters I would expect it to look like this:

```d
/**
 *  buf  = an OutBuffer containing the DDoc
 *  from = the index within `buf` to start counting from, inclusive
 *  to   = the index within `buf` to stop counting at, exclusive
 */
```

Reply:
Fixed.